### ISSUE TITLE
seafile error message

### ISSUE THREAD

**Issue by TI-Admin** [haiwen/seafile#1759]
seafile log files (5.1.4 on Ubuntu 14.04) are full of error messages

```
[09/16/2016 10:38:47 AM] ../common/seaf-db.c(483): Error prepare statement INSERT INTO RepoTokenPeerInfo VALUES (?, ?, ?, ?, ?, ?): Column count doesn't match value count at row 1.
[09/16/2016 10:40:41 AM] ../common/seaf-db.c(483): Error prepare statement UPDATE RepoTokenPeerInfo SET peer_ip=?, sync_time=?, client_ver=? WHERE token=?: Unknown column 'client_ver' in 'field list'.
[09/16/2016 10:48:48 AM] ../common/seaf-db.c(483): Error prepare statement INSERT INTO RepoTokenPeerInfo VALUES (?, ?, ?, ?, ?, ?): Column count doesn't match value count at row 1.
[09/16/2016 10:58:50 AM] ../common/seaf-db.c(483): Error prepare statement INSERT INTO RepoTokenPeerInfo VALUES (?, ?, ?, ?, ?, ?): Column count doesn't match value count at row 1.
```

**Comment by killing** [comment]
Perhaps you failed or forgot to run the upgrade script for 5.1? You can fix your database schema,

alter table RepoTokenPeerInfo add client_ver VARCHAR(20);

**Comment by TI-Admin** [comment]
Thanks. I added the missing table column.
Since I missed the upgrade from 5.0 to 5.1 I have also missed to execute seafile-server-latest/upgrade/sql/5.1.0/mysql/seahub.sql
Can I still execute upgrade_5.0_5.1.sh though already running 5.1.4 to fix the missing 5.0 to 5.1 step ?
